src/pages/Naming.tsx

```diff
@@ -0,0 +1,111 @@
+import { useForm, Controller } from 'react-hook-form'
+import React from 'react'
+import Dropdown from '../components/dropDown'
+type FormValues = {
+  language: string
```

Comment:
language: string でも良いけどもっと厳格に型付けした方が良いよ〜〜
```suggestion
  language: 'C' | 'Java' | 'Go'
```

kinds, export も同様